gulp/verifier.js

```diff
@@ -174,10 +206,43 @@ const isCheckDisabled = function(name) {
     return checks.indexOf(name) >= 0;
 };
 
-const compareKeys = function(a, b) {
+const checkStructure = function(a, b, url) {
     const aKeys = Object.keys(a).sort();
     const bKeys = Object.keys(b).sort();
-    return JSON.stringify(aKeys) === JSON.stringify(bKeys);
+    log.info(c.bold(`Structure-result for ${url}:`));
+    if (JSON.stringify(aKeys) === JSON.stringify(bKeys)) {
```

Comment:
this is good. One caveat is that maybe we want to check that all the keys are in there, but not that the stringify is equal, in case the order of the keys change at some point or something.

Reply:
Mm, yes that is true, it may happen... Fixing...

Reply:
But they are sorted before, so should be ok?

Reply:
oh. Wow I'm dumb.
You are absolutely right, sorry